src/plugin/systems/worlds.rs

```diff
@@ -0,0 +1,111 @@
+//! Systems responsible for interfacing our Bevy components with the Rapier physics engine.
```

Comment:
Thanks for putting those in a separate file :+1: The comment is not correct but I'll look into the rest of the code for my own multi-world support attempt :)